src/components/victory-chart.jsx

```diff
@@ -685,24 +685,16 @@ class VictoryChart extends React.Component {
   }
 
   render() {
-    if (this.props.containerElement === "svg") {
-      const style = this.style.base;
-      return (
-        <svg style={{ width: style.width, height: style.height, overflow: "visible" }}>
-          {this.drawAxis("x")}
-          {this.drawAxis("y")}
-          {this.drawData()}
-        </svg>
-      );
-    } else {
-      return (
-        <g>
-          {this.drawAxis("x")}
-          {this.drawAxis("y")}
-          {this.drawData()}
-        </g>
-      );
-    }
+    const style = this.style.base;
+    const props = this.props.containerElement === 'svg' ? {
+      style: { width: style.width, height: style.height, overflow: 'visible' }
+    } : {};
```

Comment:
I wonder how bad it would be to just always pass down the same props? I assume theres a good reason the `<g>` shouldn't get this same `style` attribute as the `<svg>`?